installer_examples_test.go

```diff
@@ -0,0 +1,130 @@
+package install_test
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/go-version"
+	install "github.com/hashicorp/hc-install"
+	"github.com/hashicorp/hc-install/build"
+	"github.com/hashicorp/hc-install/fs"
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/hashicorp/hc-install/src"
+)
+
+// ExampleInstaller_singleExactVersion demonstrates installation of a single exact version
```

Comment:
```suggestion
// Installation of a single exact version
```
I know it's the Go convention to start function comments with the function name, but it makes the docs harder to read:
<img width="660" alt="CleanShot 2023-02-08 at 10 45 47@2x" src="https://user-images.githubusercontent.com/45985/217494467-a8a2a65b-fde6-48d5-b19a-3b16bb01e755.png">

I find it much easier to process without the long (and maybe in that context confusing) function name:
<img width="461" alt="CleanShot 2023-02-08 at 10 46 08@2x" src="https://user-images.githubusercontent.com/45985/217494906-1bc1c910-8ee2-4cf4-8848-de911aba4950.png">

What do you think?

Reply:
Agreed, the suggested wording looks better 👍🏻